apps/src/StudioApp.js

```diff
@@ -360,6 +362,14 @@ StudioApp.prototype.init = function(config) {
     this.handleUsingBlockly_(config);
   }
 
+  var vizResizeBar = document.getElementById('visualizationResizeBar');
+  if (vizResizeBar) {
+    vizResizeBar.addEventListener('mousedown',
```

Comment:
Will add touch support in future. It is not at all discoverable without hover, but we will still do it for consistency.